storage/templates.go

```diff
@@ -27,3 +27,8 @@ func (db *store) GetTemplate(id, userID int64) (*entities.Template, error) {
 	err := db.Where("user_id = ? and id = ?", userID, id).Find(template).Error
 	return template, err
 }
+
+// DeleteTemplate deletes the template with given template id and user id from db
+func (db *store) DeleteTemplate(templateID int64, userID int64) error {
+	return db.Where("user_id = ? and id = ?", userID, templateID).Delete(entities.Template{Model: entities.Model{ID: templateID}, UserID: userID}).Error
```

Comment:
@gudgl why are we here initializing structure? we can  simple pass params?

Reply:
Refer to the delete methods for the other resources, make sure we are consistent with this.